core/src/test/groovy/com/novoda/gradle/release/LibProjectFactory.groovy

```diff
@@ -1,53 +0,0 @@
-package com.novoda.gradle.release
-
-import org.gradle.api.Project
-import org.gradle.testfixtures.ProjectBuilder
-
-final class LibProjectFactory {
```

Comment:
`shame-less plug warning:` as part of the work for the new shiny `static-analysis-plugin` we introduced a (quite extensive) [set](https://github.com/novoda/spikes/blob/master/static-analysis-plugin/plugin/src/test/groovy/com/novoda/test/TestProject.groovy) of [utilities](https://github.com/novoda/spikes/blob/master/static-analysis-plugin/plugin/src/test/groovy/com/novoda/test/TestProjectRule.groovy) that can be used to create synthetic Gradle projects in integration tests for plugins. We can apply a similar pattern here and bring back the integration tests.